### ISSUE TITLE
problem DROP TABLE FORCE doesn't clean up the disk

### ISSUE THREAD

**Issue by DanRoscigno** [StarRocks/starrocks#41046]
Someone left feedback through the feedback form in the docs that `DROP TABLE FORCE` doesn't clean up the disk.

**Comment by DanRoscigno** [comment]
closing, duplicate of #39709

**Comment by DanRoscigno** [comment]
Sorry, reopening as the other issue is Iceberg specific.

**Comment by DanRoscigno** [comment]
Please do not close this without checking to see if there is a problem.

**Comment by DanRoscigno** [comment]
keep open

**Comment by murphyatwork** [comment]
that's true, i ran into the same issue. Then i tried to change several parameters like `tablet_rowset_stale_sweep_time_sec`, but it still occupy the disk space
